multicore-sys-monitor@ccadeptic23/info.json

```diff
@@ -1 +1 @@
-{"author": "ccadeptic23"}
```

Comment:
@jaszhix: You should save the original author into a new key called *"original_author"*.